### DISCUSSION TITLE
[Feature]Mobile upload automatically creates an album

### DISCUSSION THREAD

**Discussion by danielmiao** [immich-app/immich#5258] [Feature Request]
### The feature

Want to add function: When selecting album synchronization on the mobile side, the server side can automatically create the corresponding album and join, or add the photo to the existing album with the same name.

### Platform

- [X] Server
- [ ] Web
- [X] Mobile

**Comment by shenlong-tanwen** [comment]
Marking it as duplicate in favor of #1678
